src/CalculatorWADTests/ScenarioStandard.cs

```diff
@@ -0,0 +1,160 @@
+//******************************************************************************
+//
+// Copyright (c) 2017 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium.Appium.Windows;
+using System.Threading;
+using System;
+
+namespace CalculatorTest
+{
+    [TestClass]
+    public class ScenarioStandard : CalculatorSession
+    {
+        private static WindowsElement header;
+        private static WindowsElement calculatorResult;
+
+        [TestMethod]
+        public void Addition()
+        {
+            Thread.Sleep(5000);
```

Comment:
I agree with Matt. There are a lot of sleeps in this code. Are there no events that we can have a callback and timeout for? This test suite will run about 3 minutes because of all of the sleeps. 

---
In reply to: [271839710](https://github.com/Microsoft/calculator/pull/411#discussion_r271839710) [](ancestors = 271839710)

Reply:
Removed the excessive sleep timers, original intent was to make sure stability on slower hosted agent machines.